deployment/cloud-run/test_routing_fixed.py

```
@@ -14,44 +15,72 @@
         print(f"{rule.rule} -> {rule.endpoint}")
     
     print("\n=== Testing specific endpoints ===")
     
-    # Check if root endpoint exists
-    root_routes = [rule for rule in app.url_map.iter_rules() if rule.rule == '/']
+    # Check if root endpoint exists using regex pattern for robustness
+    root_pattern = re.compile(r'^/?/?$')  # Matches '/', '//', or '' (empty string) - more flexible
+    root_routes = [rule for rule in app.url_map.iter_rules() if root_pattern.match(rule.rule)]
     if root_routes:
-        print("✅ Root endpoint (/) exists")
+        print("Root endpoint (/) exists")
```

Ruff (0.12.2)

`print` found

Remove `print`

(T201)